src/ProductView/FilterBar/FilterBar.js

```diff
@@ -1,14 +1,23 @@
 import React, { Component } from 'react';
-import InputField from '../../Utils/InputField/InputField.js';
+import RemovableInputField from '../../Utils/RemovableInputField.js';
+
 
 class FilterBar extends Component {
+    decideOnRemoveButton(currentFilter) {
+        if(this.props.filter[this.props.filter.length - 1] === currentFilter) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS1126)